scripts/lustre

```diff
@@ -8,6 +8,7 @@
 # Initialization:
 : ${OCF_FUNCTIONS_DIR=${OCF_ROOT}/lib/heartbeat}
 . ${OCF_FUNCTIONS_DIR}/ocf-shellfuncs
+RESOURCE_DIR=$(dirname $0)
```

Comment:
I don't know exactly what the right place for these extra script would be, but this is not where they should go.  /usr/lib/ocf/resource.d/* are for resource programs only.